tests/test__routines.py

```diff
@@ -44,7 +44,6 @@ def test_routine_0(glue_database, glue_table, path, use_threads, concurrent_part
     df = pd.DataFrame({"c1": [None, 1, None]}, dtype="Int16")
     wr.s3.to_parquet(
         df=df,
-        path=path,
```

Comment:
Since `path` is now optional, it was removed from some tests to check that they still pass even without referencing it